tests/plugins/tableresize/manual/iebordercollapse.md

```diff
@@ -0,0 +1,26 @@
+@bender-ui: collapsed
+@bender-tags: bug, 2823, 4.14.0
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tableresize, basicstyles, undo
```

Comment:
Can you add source view here so one can preview and experiment with table HTML/attributes?